integration/container/health_test.go

```diff
@@ -22,27 +20,17 @@ func TestHealthCheckWorkdir(t *testing.T) {
 	ctx := context.Background()
 	client := request.NewAPIClient(t)
 
-	c, err := client.ContainerCreate(ctx,
-		&container.Config{
-			Image:      "busybox",
-			Tty:        true,
-			WorkingDir: "/foo",
-			Cmd:        strslice.StrSlice([]string{"top"}),
-			Healthcheck: &container.HealthConfig{
-				Test:     []string{"CMD-SHELL", "if [ \"$PWD\" = \"/foo\" ]; then exit 0; else exit 1; fi;"},
-				Interval: 50 * time.Millisecond,
-				Retries:  3,
-			},
-		},
-		&container.HostConfig{},
-		&network.NetworkingConfig{},
-		"healthtest",
-	)
-	require.NoError(t, err)
-	err = client.ContainerStart(ctx, c.ID, types.ContainerStartOptions{})
-	require.NoError(t, err)
+	cID := container.Run(t, ctx, client, container.WithName("healthtest"), func(c *container.TestContainerConfig) {
+		c.Config.Tty = true
+		c.Config.WorkingDir = "/foo"
+		c.Config.Healthcheck = &containertypes.HealthConfig{
```

Comment:
We could create a builder for that 👼 (and function for `WorkingDir` and `Tty` 😛)

Reply:
Added WithWorkingDir and WithTty. Maybe we could defer health check later? It is quite lengthy  .